apps/mull-ui-e2e/src/integration/US-1.1.spec.ts

```diff
@@ -38,8 +38,33 @@ frameSizes.forEach((frame) => {
     it('should type into the event description input', () => {
       cy.get('#description').type('test description').should('have.value', 'test description');
     });
-    it('should type into the event location input', () => {
-      cy.get('#location').type('test location').should('have.value', 'test location');
+
+    it('should type into the event location modal and return autocompleted address', () => {
+      cy.get('#location').click();
+      cy.get('#location-input-field').should('be.visible');
+      const res = '845 Rue Sherbrooke';
```

Comment:
This is not being used anywhere
```suggestion
      const res = '845 Rue Sherbrooke';
```